core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1014,12 +1044,24 @@ paths:
             'application/json':
               schema:
                 $ref: '#/components/schemas/WorkspaceMigrationMetadata'
+        400:
+          description: Some or all workspaces in billing project were ineligible for migration.
```

Comment:
This endpoint is just for a single workspace, so it shouldn't be checking other workspaces in the billing project.

Reply:
Yup, copy/paste error.